src/compute-client/src/controller/replicated.rs

```diff
@@ -116,28 +171,58 @@ where
             // Command from controller to forward to replica.
             command = command_rx.recv() => match command {
                 None => {
-                    // Controller is no longer interested in this replica. Shut
-                    // down.
-                    return Ok(())
+                    // Controller is no longer interested in this replica. Shut down.
+                    bail!("command_rx received None")
```

Comment:
I made this case an error, now that we send correctly `DropInstance`, this should never happen.

Reply:
:+1: